shopify_python/google_styleguide.py

```diff
@@ -228,3 +237,15 @@ def __use_simple_list_comp(self, node):  # type: (astroid.ListComp) -> None
         """List comprehensions are okay to use for simple cases."""
         if len(node.generators) > 1:
             self.add_message('complex-list-comp', node=node)
+
+    def __use_cond_expr(self, node):  # type: (astroid.If) -> None
+        """Only one liner conditional expressions"""
+
+        if (len(node.body) == 1) and (len(node.orelse) == 1):
+            try:
+                if_body_name = node.body[0].targets[0].name
+                else_body_name = node.orelse[0].targets[0].name
+            except AttributeError:
```

Comment:
`node.body[0].targets[0]` is an `AssignName` object, and `.name` would raise an exception on all other object types (Such as `If`, `Assign`, etc). So I only want this to work for an `AssignName` object. 

I'm not sure how accurate this article is, https://hynek.me/articles/hasattr/, but it does talk about the differences the way this function is implemented in versions 2 & 3 of Python, and so far my luck with `hasattr`  and `isinstance` has not been very nice :( What do you think?